src/i18n.js

```diff
@@ -208,6 +208,35 @@ const resources = {
       select: 'Вибір мови',
     },
   },
+  'pl-PL': {
+    translation: {
+      title: 'Karta WiFi',
+      'desc.use':
+        'Wydrukuj prostą kartę z danymi logowania do sieci Wi-Fi. Przyklej go do lodówki, trzymaj w portfelu, itp.',
+      'desc.privacy':
+        'Twoje informacje o Wi-Fi nigdy nie są wysyłane na serwer. Na tej stronie nie stosuje się śledzenia, analiz ani odcisków palców. Zobacz',
+      'desc.source': 'kod źródłowy',
+      'wifi.login': 'Logowanie do WiFi',
+      'wifi.size': 'Rozmiar',
```

Comment:
it is added because it is used in feature https://github.com/bndw/wifi-card/pull/89